### DISCUSSION TITLE
General data question

### DISCUSSION THREAD

**Discussion by libnine** [airbnb/visx#1689] [General]
Anyone have any idea how much data a scatterplot or line chart would be able to handle? Like if I needed to grab 50k rows of data from a SQL db, would these plots be able to handle it?

**Comment by williaster** [comment]
Typically `svg`-based visualizations (which is what `visx` uses for implementation) can only handle a couple thousand elements at best. Beyond this you typically need to use a `canvas` or `web-gl` based approach.

**Reply by williaster** [reply]
I guess a line chart would be different because 50k rows could translate into a single line. but 50k data points would probably crash your browser.

**Reply by libnine** [reply]
Chart.js uses canvas, right?

**Reply by williaster** [reply]
not sure, I'd check their docs
